### PR TITLE
3209 - Add flatbed option

### PR DESCRIPTION
**Pull request by matthopson**
Screen Shot 2019-12-27 at 8 19 22 AM

Screen Shot 2019-12-27 at 8 19 32 AM

### CODE REVIEW

File: shared/src/persistence/dynamsoft/getScannerInterface.js [hunk 1]
```diff
@@ -163,8 +163,8 @@ exports.getScannerInterface = () => {
       DWObject.IfShowIndicator = false;
       DWObject.IfShowProgressBar = false;
       DWObject.Resolution = 300;
-      DWObject.IfFeederEnabled = true;
-      DWObject.IfDuplexEnabled = duplexEnabled;
+      DWObject.IfDuplexEnabled = !!(scanMode === 'duplex');
+      DWObject.IfFeederEnabled = !!(scanMode === 'feeder');
```
Comment (kkoskelin) [inline shared/src/persistence/dynamsoft/getScannerInterface.js]: nothing "wrong" with this code per se, but I'm wondering if triple-equals is EVER possible to return anything other than a true boolean of true or false.  Is the `!!` boolean coercion necessary?

Comment (matthopson) [reply shared/src/persistence/dynamsoft/getScannerInterface.js]: I was just looking at this thinking, "Kris is going to say something about this..." :P

File: web-client/src/utilities/getScanModeLabel.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/**
+ * returns a formatted scan mode label based on the scanMode value
+ *
+ * @param {object} providers the providers object
+ * @param {Function} providers.scanMode the scan mode
+ * @returns {string} the prettified scan mode label
+ *
+ */
+
+export default scanMode => {
```
Comment (kkoskelin) [inline web-client/src/utilities/getScanModeLabel.js:10]: We should consider putting the values & labels into an array of constants in `applicationContext`, and do this calculation within a computed so this function is no longer invoked from jsx?

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/utilities/getScanModeLabel.js (hunk 2 above)
Comment (matthopson) [reply web-client/src/utilities/getScanModeLabel.js:10]: Thanks! Refactor coming.

Comment (matthopson) [reply web-client/src/utilities/getScanModeLabel.js:10]: https://github.com/flexion/ef-cms/pull/3979

**Pull request merged**